x-pack/plugins/observability_solution/dataset_quality/public/components/flyout/fields_list.tsx

```diff
@@ -22,13 +22,15 @@ export function FieldsList({
   title,
   fields,
   actionsMenu: ActionsMenu,
+  dataTestSubj = `datasetQualityFlyoutFieldsList-${title.toLowerCase().split(' ').join('_')}`,
 }: {
   title: string;
   fields: Array<{ fieldTitle: string; fieldValue: ReactNode; isLoading: boolean }>;
   actionsMenu?: ReactNode;
+  dataTestSubj?: string;
```

Comment:
Does keeping it as input here serve any purpose? It could be local `const` otherwise.

Reply:
The fieldList is a reusable component which can be used in various places, i kept it as a prop